backend/app/migrator/mapper_test.go

```diff
@@ -0,0 +1,76 @@
+package migrator
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUrlMapper_URL(t *testing.T) {
```

Comment:
There is one more typical patter we want to be covered with test - change protocol only, i.e. http://something to https://something. Probably it's working fine, but would be nice to add a test case for this